pkg/operation/botanist/botanist.go

```diff
@@ -225,6 +225,10 @@ func New(ctx context.Context, o *operation.Operation) (*Botanist, error) {
 	if err != nil {
 		return nil, err
 	}
+	o.Shoot.Components.Logging.ShootEventLogger, err = b.DefaultShootEventLogger()
```

Comment:
```suggestion
	o.Shoot.Components.Logging.EventLogger, err = b.DefaultEventLogger()
```
EventLogger is already in the `Shoot` section, so we could shorten its name.